benchmark/benchmark/config.py

```diff
@@ -155,6 +155,7 @@ class Committee:
                     "primary_to_primary": x.x.x.x:x,
                     "worker_to_primary": x.x.x.x:x,
                 },
+                "network_key: NETWORK_KEY==
```

Comment:
:)

Reply:
Oh yeah, I think that's a bit random haha.

Reply:
Yes, this should be merged right before the other one.